custom/icds_reports/views.py

```diff
@@ -1815,6 +1816,16 @@ def get(self, request, *args, **kwargs):
         return JsonResponse({'message': 'Connection Successful'})
 
 
+@location_safe
```

Comment:
How is this view location safe? And more generally, how are you controlling access to this view?

Reply:
I made it to be only accessed when user is logged in. Do you think it needs more protection as this is not present at UI to be found by users?

Reply:
Made the API accessible under feature flag

Reply:
This still isn't location safe. I would remove the decorator.

Reply:
I missed it. Removed now.